jcache/commons/src/test/java/org.infinispan.jcache/AbstractTwoCachesAnnotationsTest.java

```diff
@@ -0,0 +1,108 @@
+package org.infinispan.jcache;
```

Comment:
Same thing happens with many of the other files. I've been bitten with this problem before and it causes problems with IDEs in particular.